runner.go

```diff
@@ -3,31 +3,30 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 )
 
 type runParams struct {
 	Tags          []string `json:"tags,omitempty"`
 	SmartFolderID int      `json:"smart_folder_id,omitempty"`
+	SiteID        int      `json:"site_id,omitempty"`
 }
 
 type runResponse map[string]interface{}
 
-type flagParser interface {
-	StringSlice(string) []string
-	Int(string) int
-}
-
-func createRun(f flagParser) {
-	params := makeParams(f)
+func createRun() {
+	params := makeParams()
 	resBody := postRun(params)
 
 	fmt.Println(resBody)
 }
 
-func makeParams(c flagParser) *runParams {
+func makeParams() *runParams {
+	slicedTags := strings.Split(tags, ",")
```

Comment:
no, it doesn't work that way as it is now. something to work on, or not a priority?